trin-history/src/validation.rs

```diff
@@ -106,6 +107,10 @@ impl Validator<HistoryContentKey> for ChainHistoryValidator {
                 }
                 Ok(())
             }
+            _ => {
```

Comment:
A catch-all here seems likely to bite us later. It looks like it would be an improvement to specifically include the accumulators, so that we notice when a new un-handled enum type is added.